src/components/FridayOptions/DropZone.jsx

```diff
@@ -1,17 +1,18 @@
-import { useDrop } from 'react-dnd';
-import './style.css';
-import { zones } from './defaultOptions.json';
+import { useDrop } from "react-dnd";
+import "./style.css";
+import { options, zones } from "./defaultOptions.json";
 
-const DropZone = ({ zone, onDrop }) => {
-  const getItemValues = type => zones.find(zone => type === zone.type);
+const DropZone = ({ zone, value, onDrop }) => {
```

Comment:
so ive added this minor feature to your project, get here the current value so im able to render the current value for the drop zone